src/feature/Users/Users.module.css

```
@@ -9,7 +9,7 @@
   display: flex;
   align-items: center;
   gap: 4px;
-  background: var(--primary-blue-10);
+  background: var(--primary-blue-2);
```

Update this to `--primary-blue-10`;

```suggestion
  background: var(--primary-blue-10);
```